config/versions/v6_test.go

```diff
@@ -0,0 +1,52 @@
+package versions
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/buger/jsonparser"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v6 "github.com/thrasher-corp/gocryptotrader/config/versions/v6"
+)
+
+var testData = []byte(`{
+	"portfolioAddresses": {
+	  "addresses": [
+	   {
+		"Address": "1JCe8z4jJVNXSjohjM4i9Hh813dLCNx2Sy",
+		"CoinType": "BTC",
+		"Balance": 0.00108832,
+		"Description": "",
+		"WhiteListed": false,
+		"ColdStorage": false,
+		"SupportedExchanges": ""
+	   }
+	  ]
+	}
+}`)
+
+func TestUpgrade(t *testing.T) {
```

Comment:
Sorry; We missed this.
```suggestion
func TestVersion6Upgrade(t *testing.T) {
```

Will driveBy as part of #1864